### ISSUE TITLE
Upgrade k8s to version 1.6

### ISSUE THREAD

**Issue by ant8e** [exoscale/multi-master-kubernetes#41]
It would be nice to have the lastest k8s version

**Comment by FaKod** [comment]
What does not work if you update k8s_version to 1.6 and create a cluster (never tested it)?

**Comment by retrack** [comment]
well first it k8s 1.6 uses etcd3, and CoreOS does not come with it. This would need to be sorted prior to bring up the masters.

**Comment by FaKod** [comment]
true (forgot that :-) )

So, has anyone tried this (using --storage-backend flag)? https://github.com/rook/coreos-kubernetes/commit/a3e880bf3c880e4e4551fbba54d21cf6833ffb19

The other solution is to startup a etcd v3 container (https://coreos.com/blog/toward-etcd-v3-in-container-linux.html), right?

**Comment by retrack** [comment]
for etcd3, we are waiting on CoreOS for full support as it is written: "For evaluation purposes" However a branch for experimental k8s 1.6 support would be an idea.

**Comment by ant8e** [comment]
I was under the impression that currently the playbook deploys it's own etcd cluster for k8s usage that is disticnt from the CoreOS one. But maybe I'm mistaken

**Comment by FaKod** [comment]
#45 creates a K8s cluster version v1.6.4_coreos.0 using the old etcd version 2 cluster.
Maybe this is good enough for tests, at least.

**Comment by llambiel** [comment]
This repository has been deprecated and is no longer maintained
